### PR TITLE
MGMT-8601 - Add onprem periodic and presubmit CI jobs

### PR DESCRIPTION
**Pull request by eliorerz**
/cc @osherdp

### CODE REVIEW

File: ci-operator/jobs/openshift/assisted-test-infra/openshift-assisted-test-infra-master-periodics.yaml [hunk 1]
```diff
@@ -771,6 +771,76 @@ periodics:
...
+  extra_refs:
+  - base_ref: master
+    org: openshift
+    repo: assisted-test-infra
+  labels:
+    ci-operator.openshift.io/cloud: packet
+    ci-operator.openshift.io/cloud-cluster-profile: packet-assisted
+    ci.openshift.io/generator: prowgen
+    pj-rehearse.openshift.io/can-be-rehearsed: "true"
+  name: periodic-ci-openshift-assisted-test-infra-master-e2e-metal-assisted-onprem-periodic
```
Comment (osherdp) [inline ci-operator/jobs/openshift/assisted-test-infra/openshift-assisted-test-infra-master-periodics.yaml:787]: can you add the slack notification?

**Review by osherdp — CHANGES_REQUESTED** [review]

**Review by osherdp — APPROVED** [review]

**Comment by eliorerz** [comment]
@osherdp Any idea why the onprem jobs are failing with this error?
`AttributeError: module 'yaml' has no attribute 'FullLoader'`

**Comment by osherdp** [comment]
> @osherdp Any idea why the onprem jobs are failing with this error? `AttributeError: module 'yaml' has no attribute 'FullLoader'`

I see nowhere that we deliberately install PyYAML, so that's a gap for sure
Not sure though though what has changed or why we used full-loader and not safe-loader

**Review by osherdp — APPROVED** [review]

**Pull request merged**